packages/saber/vue-renderer/app/router.js

```diff
@@ -35,6 +36,7 @@ export default () => {
     new Router({
       mode: 'history',
       routes,
+      base: publicUrl,
```

Comment:
if i remember correctly, `base` can only be a pathname?

Reply:
I tried it with an URI and that worked